apps/editor.planx.uk/src/pages/FlowEditor/components/NewSettings/TeamSettings/IntegrationSettings/SubmissionEmails/queries.ts

```diff
@@ -0,0 +1,34 @@
+import { gql } from "@apollo/client";
+
+export const GET_TEAM_SUBMISSION_INTEGRATIONS = gql`
+  query GetTeamSubmissionIntegrations($team_id: Int!) {
+    submissionIntegrations: submission_integrations(
+      where: { team_id: { _eq: $team_id } }
+    ) {
+      teamId: team_id
+      submissionEmail: submission_email
+      defaultEmail: default_email
+    }
+  }
+`;
+
+export const CREATE_TEAM_SUBMISSION_INTEGRATIONS = gql`
+  mutation InsertSubmissionIntegration {
+    insert_submission_integrations(
+      objects: { submission_email: $submission_email, team_id: $team_id }
+    ) {
+      returning {
+        emailId: email_id
+      }
+    }
+  }
+`;
+
+export const UPDATE_TEAM_SUBMISSION_INTEGRATIONS = gql`
+  mutation UpdateSubmissionIntegration {
+    update_submission_integrations_by_pk(pk_columns: { email_id: $email_id }) {
+      submissionEmail: submission_email
+      teamId: team_id
+    }
+  }
+`;
```

Comment:
When we have the UI figured out, we probably want an "upset" pattern which would allow an insert or update to be accepted.

Docs: https://hasura.io/docs/2.0/mutations/postgres/upsert/#update-selected-columns-on-conflict